resources/fathead-overview.md

```diff
@@ -6,39 +6,37 @@ Developing a Fathead Instant Answer entails writing a program that generates an
 
 The **output.txt** file that is generated will be consumed by the DuckDuckGo backend, cleaned up (details below) and then finally entered into an SQL database.
 
-
 ## Fathead Directory Structure
 
 Each Fathead Instant Answer has its own directory, which contains several files. **The name of the fathead directory must match the ID associated with the Instant Answer**.
 
-Here is an overview of the files typically found in the Fathead directory:
-
-- `lib/fathead/fathead_id/fetch.sh` &ndash; a shell script called to fetch the data.
-
-- `lib/fathead/fathead_id/download/` &ndash; a directory to hold temp files created by fetch.sh
-
-- `lib/fathead/fathead_id/parse.xx` &ndash; the script used to parse the data once it has been fetched. .xx can be `.pl`, `.py`, `.rb`, or `.js`.
+Here is an overview of the files, and directories typically found in the Fathead directory (`lib/fathead/fathead_id/`):
 
-- `lib/fathead/fathead_id/parse.sh` &ndash; a shell script wrapper around parse.xx
+Filename          | Description
+----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+download/         | A directory to hold temp files created by fetch.sh
+fetch.sh          | A shell script called to fetch the data
+parse.xx          | the script used to parse the data once it has been fetched. .xx can be `.pl`, `.py`, `.rb`, or `.js`
+parse.sh          | A shell script wrapper around parse.xx
+output.txt        | The output file. This will be automatically deployed to the beta from your Pull Request
```

Comment:
Perhaps make it really clear that they don't need to create this file, e.g.:

`The output file. This will be automatically generated from your Pull Request — you should not create it manually.`